mobile-app/cypress/integration/functional/wallet/loans/loans.spec.ts

```diff
@@ -585,39 +594,59 @@ context('Wallet - Loans Payback Non-DUSD Loans', () => {
     cy.getByTestID('text_resulting_loan_amount').contains('0.00000000')
     cy.getByTestID('text_resulting_loan_amount_suffix').contains('dTU10')
     cy.getByTestID('text_vault_id').contains(vaultId)
-    // cy.getByTestID('text_current_collateral_ratio').contains('N/A')
+    cy.getByTestID('text_current_collateral_ratio').contains('N/A')
     cy.getByTestID('button_confirm_payback_loan').click().wait(4000)
     cy.getByTestID('txn_authorization_description')
       .contains('Paying 12.00000000 dTU10')
     cy.closeOceanInterface()
     cy.wait(3000)
-    cy.checkVaultTag('READY', VaultStatus.Ready, 'vault_card_1_status', walletTheme.isDark)
+    cy.checkVaultTag('READY', VaultStatus.Ready, 'vault_card_0_status', walletTheme.isDark)
```

Comment:
i didnt run the test myself, the `vault_card` sequence will remain consistent throughout repeated test right? (just want make sure it is not flakiness)

Reply:
The reason it is `1` before is because we take another dTU10 loan to pay for the previous loan. Since the `Top up loan tokens` are fixed now, there's no need to change vault card index since there would always be just one vault for this test